src/spring/azext_spring/_build_service.py

```diff
@@ -27,6 +31,35 @@ def _update_default_build_agent_pool(cmd, client, resource_group, name, build_po
             resource_group, name, DEFAULT_BUILD_SERVICE_NAME, DEFAULT_BUILD_AGENT_POOL_NAME, agent_pool_resource)
 
 
+def create_build_service(cmd, client, resource_group, service, disable_build_service=False,
+                         registry_server=None, registry_username=None, registry_password=None):
+    if not disable_build_service:
```

Comment:
I think it is better to use opposite logic for if statement for one less level of indentation.
i.e., if disable_build_service: return